plugins/mistral/manifest.json

```diff
@@ -0,0 +1,135 @@
+{
+  "id": "mistral",
+  "description": "Mistral Content Moderation classifier leverages the most relevant policy categories for effective guardrails and introduces a pragmatic approach to LLM safety by addressing model-generated harms such as unqualified advice and PII",
+  "credentials": {
```

Comment:
"model" is not mentioned in this schema, but is referenced in the plugin code

Reply:
Mistral doesn't have multiple models for guardrailing, having it here might pass give wrong impression, just added condition for future reference (for now model is completely optional).